Automattic/vip-go-mu-plugins
# Search: Fix fatal when offloading user queries on WP 6.1

## Description
A fatal occurs when attempting to offload a user query with the default `fields` on WP 6.1:
<img width="722" alt="Screenshot 2023-01-12 at 4 10 03 PM" src="https://user-images.githubusercontent.com/16962021/212200743-08bc784c-d43c-42b9-92cc-b1476875a2ac.png">

Adapted from https://github.com/10up/ElasticPress/pull/3109
## Changelog Description

### Plugin Updated: Enterprise Search

Fix fatal when offloading user queries on WP 6.1 with the default `fields`

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1) Add the snippet:
```
 add_action( 'init', function() {
    $args = array (
        's'     => 'k',
        'ep_integrate' => true,
    );

    $authors = new WP_User_Query($args);

 });
```
2) Observe fatal in description
3) pull in PR with submodule update
4) Observe no more fatal and query being offloaded correctly
